contracts/pool/src/tests/swap.rs

```diff
@@ -756,83 +685,17 @@ fn bug_issue_169_1_percent() {
     // XYK pool formula as follows
     // Y_new = (X_in * Y_old) / (X_in + X_old)
     // Y_new = (1_000_000_000 * 110358880127) / (1_000_000_000 + 110358880127)
-    // Y_new = 991020024.637 
+    // Y_new = 991020024.637
     // Y_rnd = 991020025
     let output_amount = 991020025; // rounding
-    let fees = Decimal::percent(1) * output_amount;
-    assert_eq!(
-        result,
-        SimulateSwapResponse {
-            ask_amount: output_amount - fees,
-            spread_amount: 8979975,
-            commission_amount: fees,
-            total_return: 1000000000,
-        }
-    );
-
-    // 991020025 is the request, so 1% of that should be what's on the left hand side
-    assert_eq!(9910200, result.commission_amount);
-
-    let result = pool.simulate_reverse_swap(&token1.address, &(output_amount - fees));
-    let output_amount = 991020025i128;
-    let fees = Decimal::percent(1) * output_amount;
-    assert_eq!(
-        result,
-        SimulateReverseSwapResponse {
-            offer_amount: 1000000000i128,
-            spread_amount: Decimal::from_ratio(offer_amount, initial_liquidity) * output_amount,
-            commission_amount: fees,
-        }
-    );
-}
+    let fees;
 
-#[test]
-fn bug_issue_169_below_1_percent() {
-    let env = Env::default();
-    env.mock_all_auths();
-    env.budget().reset_unlimited();
-
-    let mut token1 = deploy_token_contract(&env, &Address::random(&env));
-    let mut token2 = deploy_token_contract(&env, &Address::random(&env));
-    if token2.address < token1.address {
-        std::mem::swap(&mut token1, &mut token2);
+    if swap_fees < 100 {
+        fees = Decimal::bps(fee_percentage) * output_amount;
+    } else {
+        fees = Decimal::percent(fee_percentage) * output_amount;
     }
```

Comment:
Keep fees in BPS, adjust parameters from test decorator. 1% is 100 etc.

```suggestion
    fees = Decimal::bps(fee_percentage) * output_amount;
```